lib/queries.js

```diff
@@ -41,117 +45,17 @@ export async function updateStatus(uri, status) {
 
 export async function countResourcesInGraph(graph) {
   const queryResult = await query(`
-      SELECT (COUNT(*) as ?count)
+      SELECT (COUNT( DISTINCT(?s)) as ?count)
       WHERE {
         GRAPH ${sparqlEscapeUri(graph)} {
-          ?s a ?o .
+          ?s ?p ?o .
```

Comment:
Isn't it almost the same but heavier ? Or you consider that a subject might not have a type ?

Reply:
Well the thing is, it might be for some reason or another not al ?s have a type. This should in theory not happen, but if it happens I want the dump to be consistent with the cache graph, so we don't have extra things to think of when debugging